index.html

```diff
@@ -41,17 +41,15 @@
           </div>
           <article class="content content--vertical">
             <div class="textBox">
-              <a
-                class="textBox--no-decoration"
-                href="http://0rz.tw/AaW9E"
-                target="_blank"
-              >
-                <h1
-                  class="textBox__head textBox__head--grey textBox__head--upper"
+              <h1>
+                <a
+                  class="textBox--no-decoration textBox__head textBox__head--grey textBox__head--upper"
+                  href="http://0rz.tw/AaW9E"
+                  target="_blank"
                 >
                   Answer your body's needs
-                </h1>
-              </a>
+                </a>
```

Comment:
What's the purpose of having a link here? Is this part of the spec? Does this benefits to the user?

Reply:
It doesn't have to be a link.